Replaces hardcoded paths with EasyBuild variables, enables HDF5
Patch based on previous versions. 
Author: J. Sassmannshausen (Imperial College London, UK)
diff --git a/vasp.6.4.3.orig/arch/makefile.include.nvhpc_omp_acc b/vasp.6.4.3/arch/makefile.include.nvhpc_omp_acc
index 8e98b4e..3014a1c 100644
--- a/vasp.6.4.3.orig/arch/makefile.include.nvhpc_omp_acc
+++ b/vasp.6.4.3/arch/makefile.include.nvhpc_omp_acc
@@ -100,10 +100,10 @@ INCS       += -I$(FFTW_ROOT)/include
 #LLIBS      += -cudalib=cusolvermp,cublasmp -lnvhpcwrapcal
 
 # HDF5-support (optional but strongly recommended)
-#CPP_OPTIONS+= -DVASP_HDF5
-#HDF5_ROOT  ?= /path/to/your/hdf5/installation
-#LLIBS      += -L$(HDF5_ROOT)/lib -lhdf5_fortran
-#INCS       += -I$(HDF5_ROOT)/include
+CPP_OPTIONS+= -DVASP_HDF5
+HDF5_ROOT   = ${EBROOTHDF5}
+LLIBS      += -L$(HDF5_ROOT)/lib -lhdf5_fortran
+INCS       += -I$(HDF5_ROOT)/include
 
 # For the VASP-2-Wannier90 interface (optional)
 #CPP_OPTIONS    += -DVASP2WANNIER90
